nc/graphics/Mesh.cpp

```diff
@@ -1,7 +1,11 @@
 #include "debug\NcException.h"
 #include "graphics\d3dresource\GraphicsResourceManager.h"
 #include "graphics\Mesh.h"
-#include "utils\objloader\ObjLoader.h"
+#include <assimp\Importer.hpp>
```

Comment:
Convention is to use <> for compiler implementation includes, so this should be in quotes.